cli/flox/src/utils/errors.rs

```diff
@@ -515,6 +515,34 @@ pub fn format_locked_manifest_error(err: &LockedManifestError) -> String {
         // already user friendly.
         // some commands catch these errors and process them separately
         // e.g. `flox pull`, `flox push`
+
+        // 116: toml parsing error, just print the error message
+        // https://github.com/flox/flox/issues/852
+        LockedManifestError::LockManifest(CallPkgDbError::PkgDbError(PkgDbError {
+            exit_code: 116,
+            context_message:
+                Some(ContextMsgError {
+                    caught: Some(caught),
+                    ..
+                }),
+            ..
+        })) => {
+            let message = &caught.message;
+            let un_prefixed = message.strip_prefix("[error] ").unwrap_or(message);
+            formatdoc! {"
```

Comment:
Question: having never used the `formatdoc!` macro, what's the purpose of using it here instead of just returning a `String`?

Reply:
Serves as a reminder to add the error prefix..

Currently, no use, its like `format!{"message"}` though clippy _is_ complaing about that...